src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineCatalog.cs

```diff
@@ -13,6 +13,29 @@ namespace Microsoft.ML
     /// </summary>
     public static class FactorizationMachineExtensions
     {
+        /// <summary>
+        /// Predict a target using a field-aware factorization machine algorithm.
```

Comment:
Please add `Note that because there is only one feature column, the underlying model is equivalent to standard factorization machine`.